.github/workflows/binary-applications.yml

```diff
@@ -47,21 +47,26 @@ jobs:
                   path: ~\AppData\Local\pip\Cache
                   key: windows-py${{ env.PYTHON_VERSION }}-${{ env.PYTHON_ARCH}}-pip-${{ hashFiles('**/requirements*.txt') }}-exe
 
-            - name: Set up python
-              uses: actions/setup-python@v1
+            - name: Setup conda environment
+              uses: goanpeca/setup-miniconda@v1.1.2
               with:
-                  python-version: ${{ env.PYTHON_VERSION }}
-                  architecture: ${{ env.PYTHON_ARCH }}
+                  activate-environment: winbin-env  # This is also hardcoded in Pyinstaller's invest.spec

```

Comment:
Isn't the spec referring to `os.environ['CONDA_PREFIX']` now? Maybe this comment is outdated.

Reply:
Yep, these were leftover comments and have been removed.